examples/tf13-compatible/complete/tamr-vm.tf

```diff
@@ -13,7 +13,7 @@ data "aws_ami" "tamr-vm" {
 }
 
 module "tamr-vm" {
-  source = "git::git@github.com:Datatamer/terraform-emr-tamr-vm?ref=4.4.0"
+  source                      = "git::git@github.com:Datatamer/terraform-emr-tamr-vm?ref=4.4.0"
```

Comment:
Let's use the latest release for the VM module which is 4.4.2.

Reply:
Fixed [here](f8d63edbb9bc7d586e2868759543046b8271a9a2)

Reply:
Fixed [here](https://github.com/Datatamer/terraform-aws-tamr-config/pull/33/commits/f8d63edbb9bc7d586e2868759543046b8271a9a2)